grammars/php.cson

```diff
@@ -2495,7 +2495,7 @@
       }
     ]
   'php_doc_types':
-    'match': '(?i)[a-z_\\x{7f}-\\x{7fffffff}\\\\][a-z0-9_\\x{7f}-\\x{7fffffff}\\\\]*([|&][a-z_\\x{7f}-\\x{7fffffff}\\\\][a-z0-9_\\x{7f}-\\x{7fffffff}\\\\]*)*'
+    'match': '(?i)\\??[a-z_\\x{7f}-\\x{7fffffff}\\\\][a-z0-9_\\x{7f}-\\x{7fffffff}\\\\]*([|&]\\??[a-z_\\x{7f}-\\x{7fffffff}\\\\][a-z0-9_\\x{7f}-\\x{7fffffff}\\\\]*)*'
```

Comment:
Sorry, for the late reply. Can you add `keyword.operator.nullable-type.php` for the `?` ? Other than that it looks good.

```coffee
          {
            'match': '\?'
            'name': 'keyword.operator.nullable-type.php'
          }
```

Reply:
Nice catch, thank you.